src/Functions/GatherUtils/Algorithms.h

```diff
@@ -461,65 +553,96 @@ inline ALWAYS_INLINE bool sliceEqualElements(const GenericArraySlice & first, co
     return first.elements->compareAt(first_ind + first.begin, second_ind + second.begin, *second.elements, -1) == 0;
 }
 
-template <bool all, typename T, typename U>
+template <typename T>
+bool insliceEqualElements(const NumericArraySlice<T> & first [[maybe_unused]],
+                          size_t first_ind [[maybe_unused]],
+                          size_t second_ind [[maybe_unused]])
+{
+    /// TODO: Decimal scale
```

Comment:
Do you plan to do something here?

Reply:
it is done already. 
Just forgot to cleanup the comment. Sorry.